.changeset/mean-queens-roll.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Fixes an issue that prevents `waitForPendingWrites()` from resolving in background tabs when multi-tab is used.
```

Comment:
Is it worth linking to the specific issue we're addressing?

Reply:
Done

Reply:
The changelog generator can automatically generate the issue link in the changelog by looking for issue links in the PR comment, so it's fine to not include it manually. No need to worry about 2 issue links - the generator will not generate one if it sees you manually included it.